### DISCUSSION TITLE
Issue when trying to change values that have been modeled as IfcPropertyListValue

### DISCUSSION THREAD

**Discussion by grndng** [IfcOpenShell/IfcOpenShell#3258] [Q&A]
Hey there! I'm currently looking into an IFC File and what I want to do is to:
- [ x ] filter for a type e.g. `IfcWallType`
- [ x ] Chose a specific Wall Type (e.g. `Outer Walls`)
- [ x ] Get a Property Sets of `Outer Walls` (e.g. `time frame of change`)
- [ x ] Get the value of `time frame of change` (which is modeled as IfcPropertyListValue)
- [  ] Set a new value for `time frame of change`

When trying to set the property with the following code snippet, I'm getting an `AttributeError: entity instance of type 'IFC4.IfcPropertyListValue' has no attribute 'NominalValue'` and I couldn't figure out a way to solve it yet.

```python
ifcopenshell.api.run(
                           "pset.edit_pset",
                            file = ifc_file, # my ifc file
                            pset = ifc_file.by_id(pset["id"]), # chosing the propertyset in my file by id
                            properties = {propname:future_value} # passing the name of the property and the future value
)
```

Please let me know if I can help with additional information or the information provided is not sufficient to answer my question. Essentially I think I need a pointer in a direction regarding the ListValues. Thank you in advance!

**Comment by aothms** [comment]
Did you see https://github.com/IfcOpenShell/IfcOpenShell/discussions/3228 https://github.com/IfcOpenShell/IfcOpenShell/pull/3237?
